lib/rails_cloudflare_turnstile/view_helpers.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RailsCloudflareTurnstile
+  module ViewHelpers
+    def cloudflare_turnstile(action: "other")
+      return nil unless RailsCloudflareTurnstile.enabled?
+      content_tag(:div, class: "cloudflare-turnstile") do
+        concat turnstile_div(action)
+      end
+    end
+
+    def cloudflare_turnstile_script_tag
+      return nil unless RailsCloudflareTurnstile.enabled?
+      content_tag(:script, :src => js_src, "async" => true, "defer" => true) do
```

Comment:
As far as I know, Having `async` and `defer` attributes at the same time on a `script` tag is problematic. Async loads the script in the background and when the script is loaded, it interrupts the page load and executes the script. However,  while `defer` is also downloading script in the background, it waits until the full page render.

Considering that cloudflare's turnstile has an important role on the page, `async` to be `true` makes more sense to me. 

Here is more information on this topic: https://pagespeedchecklist.com/async-and-defer

Reply:
The cloudflare manual recommends setting both, but if you feel strongly I can unset defer.